packages/data-service-generator/src/tests/__snapshots__/create-data-service-custom-path.spec.ts.snap

```diff
@@ -11645,18 +11645,16 @@ describe("Testing the secrets manager base class", () => {
 });
 ",
   "test/src/providers/secrets/base/secretsManager.service.base.ts": "import { ConfigService } from "@nestjs/config";
+import { EnumSecretsNameKey } from "../secretsNameKey.enum";
```

Comment:
Where is the enum file? I can't see it in the snapshot

Reply:
here: https://github.com/amplication/amplication/pull/7398/files/8fd84fcc7c4ce6d3048d72b5e217fdd30dbc27d5#diff-8e56062d3bf78d23d02b260a21d149cd721a3a1e11b453eb0cac779529ec9554R11685